Lib/test/test_import/__init__.py

```diff
@@ -80,6 +80,43 @@ def test_from_import_missing_attr_raises_ImportError(self):
         with self.assertRaises(ImportError):
             from importlib import something_that_should_not_exist_anywhere
 
+    def test_from_import_missing_attr_has_name_and_path(self):
+        """
```

Comment:
We don't include docstrings on tests (and we PEP 8 doesn't allow starting a docstring with a newline).